gigglebot.ts

```diff
@@ -172,7 +172,7 @@ namespace gigglebot {
         stripNeopixel.setPixelColor(_i, neopixel.colors(NeoPixelColors.Black))
     }
     stripNeopixel.show()
-    if (voltageRead() < 3400) {
+    if (voltageRail() < 3400) {
```

Comment:
This should be named `voltageBattery`. If the battery voltage is below 3.4v the LEDs should be red. This could be due to dead batteries, or due to the power switch being in the `OFF` position.

Reply:
[Line 765](https://github.com/DexterInd/pxt-gigglebot/blob/master/custom.ts#L765) should also be changed to `voltageBattery`.

Reply:
I thought this one was going to be giving me the Rail voltage?  I'm happy to change it

Reply:
I intend to keep `GET_VOLTAGE_BATTERY` as battery voltage, and use the new `GET_VOLTAGE_RAIL` for reading the rail voltage. Disabling the motors will still be based on battery voltage, so the eye color should still be set accordingly.